packages/fluentui/react-builder/src/components/MessageComputer.tsx

```diff
@@ -0,0 +1,247 @@
+/*
+ TODO:
+* Add the missing and not so obvious attributes (e.g. "aria-haspopup" or "aria-expanded") for the already defined roles (e.g. "menuitem" or "checkbox") according to the specification.
+* With JAWS, in the case of the element with the "listbox" role, differentiate between having and not having the aria-multiselectable="true" attribute. If this attribute is present, then aria-selected="false" on the child elements with the role "option" behave differently than if it is not present. Specifically, if aria-multiselectable="true" is present, the aria-selected="false" causes the narration of "not selected", but if present, having the aria-selected attribute makes no difference to the narration. 
+ * What about the "disabled" state?
+ */
+import SRMC from './SRMC-Definitions';
+import './SRMC-Rules-Win_JAWS';
+
+export default class MessageComputer {
+  private computedMessageParts: { [key: string]: string } = {
```

Comment:
`{ [key: string]: string } ` is an incredibly funny type :)

I would suggest you either strongly type this object or at the very least use the `Record<string, string>` utility type for better readability

Reply:
```suggestion
  private computedMessageParts: Record<string, string> = {
```

Just to make it simpler :)

Reply:
How would the "strongly typed" version look like?

Reply:
you create a type with each of those properties

```
type ComputedMessage = {
  value: string;
  name: string;
}
```

This will mean that you get some handy tools like autocomplete in your editor and some type safety. However, if those aren't important then stick with the other suggestion and use `Record`

Reply:
Thanks. What is actually bad on `{ [key: string]: string }`? It looks concise and clear to me.

Reply:
So if you do not type this object at all, typescript type inference will kick in a 'automatically' type this object for you when it's imported and you get all the benefits of declaring a type (you just can't reuse the type)

I do not like any type like `{ [key: string]: string }` or `Record<string, string>` since at that point you might as well use the type `any` or `object` to the same effect. 

why I suggest `Record` if you really really want a generic type with no information on the properties is that it's more readable

Reply:
Then again I am still not in the team yet so feel free to ask others what the team's opinions on this are :)

Reply:
Okay, thanks. But `{[key: string]: string}` also ensures type safety. It's not the same as `:any`.

Reply:
Oh, that's new to me, that's fair. Apologies.

Reply:
`Record<string, string>` in this case will be more readable, concise, and consistent with what we have.